challenge_restaurant/README.md

```diff
@@ -3,28 +3,12 @@
 Responsible: Rein
 
 ## Startup
+```
+* amigo-start should already be running
```

Comment:
Place commands between three graves.

Reply:
Fixed in d34e371aa795efb2a8317025b878fcc08ceb7968